docs/UFS_unified_variables_table.csv

```diff
@@ -148,97 +171,54 @@ Averaged precipitation advected heat flux,pah_ave,paha,2d,phy,
-,axyl1j,axyl1j,3d,dyn,aqfcmaq_on = True
-,axyl2j,axyl2j,3d,dyn,aqfcmaq_on = True
-,axyl3j,axyl3j,3d,dyn,aqfcmaq_on = True
-,pm25ac,pm25ac,3d,dyn,aqfcmaq_on = True
-,pm25at,pm25at,3d,dyn,aqfcmaq_on = True
-,pm25co,pm25co,3d,dyn,aqfcmaq_on = True
+Average ozone concentration,o3_ave,avgozcon,3d,dyn,aqf_on=True
+Average particulate matter (fine),pm25_ave,avgpmtf,3d,dyn,aqf_on=True
+Aerosol optical depth,aod,aqm_aod550,2d,phy,aqf_on=True
 Instantaneous aod550 optical depth,aod550,aod550,2d,phy,rdaod = True
-,du_aod550,du_aod550,2d,phy,rdaod = True
-,ss_aod550,ss_aod550,2d,phy,rdaod = True
-,su_aod550,su_aod550,2d,phy,rdaod = True
-,oc_aod550,oc_aod550,2d,phy,rdaod = True
-,bc_aod550,bc_aod550,2d,phy,rdaod = True
+Instantaneous aod550 optical depth (dust),du_aod550,du_aod550,2d,phy,rdaod = True
+Instantaneous aod550 optical depth (seasalt),ss_aod550,ss_aod550,2d,phy,rdaod = True
+Instantaneous aod550 optical depth (sulfates),su_aod550,su_aod550,2d,phy,rdaod = True
+Instantaneous aod550 optical depth (organic carbon),oc_aod550,oc_aod550,2d,phy,rdaod = True
+Instantaneous aod550 optical depth (black carbon),bc_aod550,bc_aod550,2d,phy,rdaod = True
+Dust 1,dust1,dt1,3d,dyn,GOCART
+Dust 2,dust2,dt2,3d,dyn,GOCART
+Dust 3,dust3,dt3,3d,dyn,GOCART
+Dust 4,dust4,dt4,3d,dyn,GOCART
+Dust 5,dust5,dt5,3d,dyn,GOCART
+Sea salt (1-5),seas1/.../seas5,salt,3d,dyn,GOCART - all requested read into 4d array
+Black carbon,bc1/bc2,soot,3d,dyn,GOCART - all requested read into 4d array
+Organic carbon,oc1/oc2,waso,3d,dyn,GOCART - all requested read into 4d array
+Sulfate,sulf,suso,3d,dyn,gocart_on=True or gccpp_on=True
+Sulfate,so4,suso,3d,dyn,nasa_on=True
+Nitrate,no3an1/no3an2/no3an3,no3,3d,dyn,GOCART - all requested read into 4d array
+NH4,nh4a,nh4,3d,dyn,GOCART
+Particulate matter (PP25),pp25,pp25,3d,dyn,gocart_on=True
+Particulate matter (PM25),pm25,pp25,3d,dyn,nasa_on=True
+Particulate matter (PP10),pp10,pp10,3d,dyn,gocart_on=True
+Particulate matter (PM10),pm10,pp10,3d,dyn,nasa_on=True
+Dust emission fluxes (1-5),duem001/.../duem005,duem,2d,phy, all requested read into 3d array
+Dust sedimentation fluxes (1-5),dust1sd/.../dust5sd,dusd,2d, phy, all requested read into 3d array
+Dust dry deposition fluxes (1-5),dust1dp/.../dust5dp,dupd,2d,phy, all requested read into 3d array
+Dust wet deposition fluxes (1-5),dust1wtl/.../dust5wtl,duwt,2d,phy, all requested read into 3d array
+Dust scavenging fluxes (1-5),dust1wtc/.../dust5wtc,dusv,2d,phy, all requested read into 3d array
+Seasalt emission fluxes (1-5),ssem001/.../ssem005,ssem,2d,phy, all requested read into 3d array
+Seasalt **emission/sedimentation** fluxes (1-5),seas1sd/.../seas5sd,sssd,2d,phy, all requested read into 3d array
+Seasalt dry deposition fluxes (1-5),seas1dp/.../seas5dp,ssdp,2d,phy,all requested read into 3d array
+Seasalt wet deposition fluxes (1-5),seas1wtl/.../seas5wtl,sswt,2d,phy,all requested read into 3d array
+Seasalt scavenging fluxes (1-5),seas1wtc,sssv,2d,phy,all requested read into 3d array

```

Comment:
@hertneky Do you mean the table should be fixed or the code should be fixed? At the moment, the code reads: 
```! retrieve seasalt scavenging fluxes
      do K = 1, nbin_ss
       if ( K == 1) VarName='seas1wtc'
       if ( K == 2) VarName='seas1wtc'
       if ( K == 3) VarName='seas1wtc'
       if ( K == 4) VarName='seas1wtc'
       if ( K == 5) VarName='seas1wtc'
      call read_netcdf_2d_para(ncid2d,ista,ista_2l,iend,iend_2u,jsta,jsta_2l,jend,jend_2u,&
      spval,VarName,chem_2d)
      sssv(1:im,jsta_2l:jend_2u,K)=chem_2d(1:im,jsta_2l:jend_2u)
       enddo```
Based on the code, seas1wct is correct for the table, but I agree that it seems odd that it's not seas1wct-seas5wct.
@kayeekayee Do you know?

Reply:
To me it seems that it should read as below otherwise it's just reading in seas1wtc for all K
do K = 1, nbin_ss
       if ( K == 1) VarName='seas1wtc'
       if ( K == 2) VarName='seas2wtc'
       if ( K == 3) VarName='seas3wtc'
       if ( K == 4) VarName='seas4wtc'
       if ( K == 5) VarName='seas5wtc'
      call read_netcdf_2d_para(ncid2d,ista,ista_2l,iend,iend_2u,jsta,jsta_2l,jend,jend_2u,&
      spval,VarName,chem_2d)
      sssv(1:im,jsta_2l:jend_2u,K)=chem_2d(1:im,jsta_2l:jend_2u)
       enddo

Reply:
Fixed!